dockerfiles/docker-compose.yml

```diff
@@ -29,7 +29,8 @@ services:
     volumes:
       - ${PWD}/common/dockerfiles/entrypoints/common.sh:/usr/src/app/docker/common.sh
       - ${PWD}/common/dockerfiles/entrypoints/proxito.sh:/usr/src/app/docker/proxito.sh
-      - ${PWD}/../readthedocs-ext:/usr/src/app/checkouts/readthedocs-ext
+      - ${PWD}/${RTDDEV_PATH_EXT:-../readthedocs-ext}:/usr/src/app/checkouts/readthedocs-ext
+      - ${PWD}/${RTDDEV_PATH_EXT_THEME:-../readthedocs-ext-theme}:/usr/src/app/checkouts/readthedocs-ext-theme
```

Comment:
What repository is `readthedocs-ext-theme`? I'd say the default should be the same name of the repository (I suppose it's https://github.com/readthedocs/ext-theme, which the default would be `ext-theme` in that case)